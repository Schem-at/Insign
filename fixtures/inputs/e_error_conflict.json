[
  { "pos": [0, 64, 0], "text": "@test=rc([0,0,0],[1,1,1])\n#test:value=1" },
  { "pos": [10, 64, 10], "text": "@other=rc([5,0,0],[6,1,1])\n#test:value=2" }
]
