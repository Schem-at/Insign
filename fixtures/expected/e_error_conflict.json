{
  "error": "Parse error: Metadata conflict for region 'test' key 'value': different values across tuples. First: 1 at tuple 0 statement 1, Conflict: 2 at tuple 1 statement 1",
  "tuple_index": null,
  "statement_index": null
}
